lib/inject/bower-files.js

```diff
@@ -84,7 +84,7 @@ module.exports = function () {
 
     function greatestCommonBase(absolute) {
       if (base) {
-        for (var i = 0; (base[i] === absolute[i]); i++);
+        for (var i = 0; (base[i] === absolute[i]); i++) {} /* jshint -W035 */
```

Comment:
Cool code. However convert this to a (do)while loop with internal increment in order to make this understandable.